client/tests/integration/smartcontracts/executor_remove_permission/src/lib.rs

```diff
@@ -25,13 +26,9 @@ struct Executor {
 pub fn migrate(_block_height: u64) -> MigrationResult {
     // Note that actually migration will reset token schema to default (minus `CanUnregisterDomain`)
     // So any added custom permission tokens will be also removed
-    let mut schema = default_permission_schema();
-    schema.remove::<iroha_executor::default::tokens::domain::CanUnregisterDomain>();
-
-    let (token_ids, schema_str) = schema.serialize();
-    iroha_executor::set_permission_schema(
-        &iroha_executor::data_model::permission::PermissionSchema::new(token_ids, schema_str),
-    );
+    DataModelBuilder::with_default_permissions()
+        .remove_permission::<CanUnregisterDomain>()
+        .set();
```

Comment:
```suggestion
        .build();
```

since it's a builder?